flaml/automl/model.py

```diff
@@ -2090,7 +2090,6 @@ def fit(self, X_train, y_train, budget=None, free_mem_ratio=0, **kwargs):
         if weight is not None:
             kwargs["sample_weight"] = weight
         self._model = model
-        self.params[self.ITER_HP] = self._model.tree_count_

```

Comment:
Let's comment this out instead of removing it.

Reply:
Done - added an explanatory comment too, linking to this PR.